kubernetes/helm_charts/secor/config/secor.common.properties

```diff
@@ -0,0 +1,495 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+############
+# MUST SET #
+############
+
+# Regular expression matching names of consumed topics.
+secor.kafka.topic_filter={{ dig $.Release.Name "topic" "" $.Values.secor_jobs }}
+secor.kafka.topic_blacklist=
+
+# Choose what to fill according to the service you are using
+# in the choice option you can fill S3, GS, Swift or Azure
+cloud.service=Azure
```

Comment:
The problem i see with this property file is how do we override values? For example this here is hardcoded to Azure. But we don't have an option to change this in private or anywhere and need to clone the repo. Same with the rest of the properties here. It would be good if we have some placeholders like `cloud.service="{{ secor_cloud_service | default('Azure')}}`